spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/authentication/provider/AccessController.java

```diff
@@ -0,0 +1,27 @@
+/*
+
+ * Copyright (C) 2020-2022 Huawei Technologies Co., Ltd. All rights reserved.
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.huaweicloud.governance.authentication.provider;
+
+/**
+ * Add black / white list control to service access
+ */
+public interface AccessController {
+  public boolean isAllowed(String serviceId, String instanceId);
```

Comment:
Maybe can think better extensions . Let service comb and nacos has minimal code

Reply:
Have change achieve